modules/javafx.graphics/src/main/java/javafx/css/StyleablePropertyFactory.java

```diff
@@ -121,12 +121,14 @@ public final class MyButton extends Button {
  * <pre><code>
  public final class MyButton extends Button {
 
+     private static CssMetaData<JavadocFixStylableProperty, Boolean> SELECTED;
```

Comment:
should it be

```
private static CssMetaData<MyButton, Boolean> SELECTED;
```

in order to compile?

Reply:
Updated this change